packages/at_chops/test/at_chops_test.dart

```diff
@@ -220,6 +220,28 @@ void main() {
       expect(decryptionResult.atEncryptionMetaData.iv, iv);
       expect(decryptionResult.result, data);
     });
+
+    test('validate decryption behaviour with null iv', () {
+      final aesKey = AtChopsUtil.generateSymmetricKey(EncryptionKeyType.aes256);
+      final atChopsKeys = AtChopsKeys()..selfEncryptionKey = aesKey;
+      final atChops = AtChopsImpl(atChopsKeys);
+
+      var iv = AtChopsUtil.generateRandomIV(15);
+      var utf8EncData = utf8.encode('abcd');
+      var encryptionResult =
+          atChops.encryptBytes(utf8EncData, EncryptionKeyType.aes256, iv: iv);
+
+      bool exceptionCatchFlag = false;
+      try {
+        atChops.decryptBytes(encryptionResult.result, EncryptionKeyType.aes256,
+            iv: null);
+      } on Exception catch (e) {
```

Comment:
check other test files how to assert exception and exception message

Reply:
addressed in [00b72de](https://github.com/atsign-foundation/at_libraries/pull/554/commits/00b72dea48a0b9cafd1d66a4178047b2e2444afa)